app/assets/javascripts/discourse/app/components/sidebar/more-section-link.hbs

```diff
@@ -1,15 +1,30 @@
-<Sidebar::SectionLink
-  @shouldDisplay={{@sectionLink.shouldDisplay}}
-  @linkName={{@sectionLink.name}}
-  @route={{@sectionLink.route}}
-  @href={{@sectionLink.href}}
-  @query={{@sectionLink.query}}
-  @title={{@sectionLink.title}}
-  @content={{@sectionLink.text}}
-  @currentWhen={{@sectionLink.currentWhen}}
-  @badgeText={{@sectionLink.badgeText}}
-  @model={{@sectionLink.model}}
-  @models={{@sectionLink.models}}
-  @prefixType={{@sectionLink.prefixType}}
-  @prefixValue={{@sectionLink.prefixValue}}
-/>
\ No newline at end of file
+{{#if @sectionLink.shouldDisplay}}
```

Comment:
Just curious, why do we need this conditional when `Sidebar::SectionLink` already takes in a `shouldDisplay` args and handles this.

Reply:
ahh, it might be not required anymore. I did it for that new-topic button which is not a case anymore